cast/src/launcher/layout/hc-cast.ts

```diff
@@ -158,16 +158,6 @@ class HcCast extends LitElement {
 
   protected updated(changedProps) {
     super.updated(changedProps);
-    if (this.castManager && this.castManager.status) {
-      const selectEl = this.shadowRoot!.querySelector("select");
```

Comment:
I can't find this `select` is this unused code or am I blind?

Reply:
correct. That's old code.